packages/build/tests/install/tests.js

```diff
@@ -110,17 +110,3 @@ test('Install local plugin dependencies: no root package.json', async (t) => {
 test('Install local plugin dependencies: missing plugin in netlify.toml', async (t) => {
   await runFixture(t, 'local_missing')
 })
-
-test('Automatically install missing plugins locally', async (t) => {
```

Comment:
Those three tests are not needed anymore because the automatic installation of non-`plugins.json` plugins has been removed, and other tests are covering the new behavior.

Reply:
🎉